zaas-client/src/main/java/org/zowe/apiml/zaasclient/service/internal/ZaasClientImpl.java

```diff
@@ -74,15 +72,25 @@ private Object getScheme(boolean httpOnly) {
 
     @Override
     public String login(String userId, String password, String newPassword) throws ZaasClientException {
-        if (userId == null || password == null || newPassword == null || userId.isEmpty() || password.isEmpty() || newPassword.isEmpty()) {
+        return login(userId, password.toCharArray(), newPassword.toCharArray());
+    }
+
+    @Override
+    public String login(String userId, String password) throws ZaasClientException {
+        return login(userId, password.toCharArray());
```

Comment:
Is there possible to have null in passwords?

```suggestion
        return login(userId, password == null ? null : password.toCharArray(), newPassword == null ? null : newPassword.toCharArray());
    }

    @Override
    public String login(String userId, String password) throws ZaasClientException {
        return login(userId, password == null ? null : password.toCharArray());
```